pkg/processor/test/suite/suite.go

```diff
@@ -453,9 +450,11 @@ func (suite *TestSuite) GetDeployOptions(functionName, functionPath string) *pla
 // function uses async trigger mode
 func (suite *TestSuite) GetDeployOptionsAsync(functionName, functionPath string) *platform.CreateFunctionOptions {
 	createFunctionOptions := suite.GetDeployOptions(functionName, functionPath)
-	trigger, found := createFunctionOptions.FunctionConfig.Spec.Triggers["http-trigger"]
-	suite.Require().True(found)
-	trigger.Mode = functionconfig.AsyncTriggerWorkMode
+	createFunctionOptions.FunctionConfig.Spec.Triggers = map[string]functionconfig.Trigger{
+		"http-trigger": {
+			Kind: "http",
+			Mode: functionconfig.AsyncTriggerWorkMode,
+		}}
```

Comment:
An http trigger can still be configured in the yaml that's in the `Spec.Build.Path`

Reply:
@TomerShor yes, but here we need to specify the mode and it can only be specified on trigger level

Reply:
I understand, but this won't fix the issue of multiple http triggers.

Reply:
@TomerShor it should, as async options are used in one test suite only